mycroft.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+if [ ! -z $1 ]
+then
+     screen -XS mycroft-service quit
```

Comment:
I'm not super familiar with what screen is doing here? Not a blocker, just want to understand what kill signals the process is going to get.